client/app/app.js

```diff
@@ -160,58 +160,19 @@ angular
             level2: "2"
         }
     })
-    .config(["$provide", function ($provide) {
+    .run((atInternet, TRACKING, OvhApiMe) => {
         "use strict";
 
-        $provide.decorator("atInternet", ($delegate, User, TRACKING) => {
-            const delegateTrackPage = $delegate.trackPage;
-            let isDefaultConfigurationSet = false;
-            let trackPageRequestArgumentStack = [];
-
-            // Decorate trackPage to stack requests until At-internet default configuration is set
-            $delegate.trackPage = (...args) => {
-                if (isDefaultConfigurationSet) {
-                    delegateTrackPage.apply($delegate, args);
-                } else {
-                    trackPageRequestArgumentStack.push(args);
-                }
-            };
-
-            // Get country from User
-            User.getUser()
-                .then((result) => {
-                    const settings = angular.copy(TRACKING.config);
-
-                    // Set identifiedVisitor ID
-                    if ($delegate.isTagAvailable() && result.nichandle) {
-                        const atinternetTag = $delegate.getTag();
-
-                        // atinternetTag.page.set();
-                        atinternetTag.identifiedVisitor.set({ id: result.nichandle });
-                        atinternetTag.dispatch();
-                    }
-
-                    // Set countryCode
-                    settings.countryCode = result.billingCountry;
-                    $delegate.setDefaults(settings);
-
-                    isDefaultConfigurationSet = true;
+        const config = TRACKING.config;
 
-                    _.forEach(trackPageRequestArgumentStack, (trackPageArguments) => {
-                        delegateTrackPage.apply($delegate, trackPageArguments);
-                    });
-                })
-                .catch(() => {
-                    // Reset trackPage
-                    $delegate.trackPage = () => false;
-                    trackPageRequestArgumentStack = [];
-
-                    $delegate.setEnabled(false);
-                });
-
-            return $delegate;
-        });
-    }])
+        OvhApiMe.Lexi().get().$promise
+            .then((me) => {
+                config.countryCode = me.country;
+                config.currencyCode = me.currency && me.currency.code;
+                config.visitorId = me.customerCode;
+                atInternet.setDefaults(config);
+            });
```

Comment:
no catch ?

Reply:
I'm not able to see what you want me to catch there.
IMO at-internet shouldn't handle a fail of /me.